### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by ShashiSubramanya**
Readme and other edits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,784 @@
...
+<br/>
+
+# ThoughtSpot Charts SDK <br/>
+
+ThoughtSpot Charts SDK allows developers to integrate custom charts into ThoughtSpot. Developers can use this SDK to create custom charts in Javascript using charting libraries such as HighCharts and upload them to ThoughtSpot.   
+
+
+# Get started
+This tutorial demonstrates how to create a Gantt chart demo using HighCharts. 
+<insert links and fiddle links above>
```
Comment (ShashiSubramanya) [inline README.md:14]: If you want to add this later, we can comment it out in this version

File: README.md [hunk 2]
```diff
@@ -0,0 +1,784 @@
...
+
+<Should node.js and npm installation be added as prereq? Any specific versions to consider?>
```
Comment (ShashiSubramanya) [inline README.md:16]: Should node.js and npm installation be added as prereq? Any specific versions to consider? If not, we can remove this text.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,784 @@
...
+            },
+        ],
+    };
+    return [chartConfig];
+},
+```
+
+> There is a better way to be able to configure the dimensions and columns in dimensions. But that is relatively a more advanced topic and not required for this demo. You can use the link below to look into the details of the same.
+>
+
```
Comment (ShashiSubramanya) [inline README.md:298]: Should we remove this text?
> There is a better way to be able to configure the dimensions and columns in dimensions. But that is relatively a more advanced topic and not required for this demo. You can use the link below to look into the details of the same.
>

**Review by ShashiSubramanya — COMMENTED** [review]
Added a few comments inline

File: README.md [hunk 4]
```diff
@@ -0,0 +1,780 @@
...
+```
+
+2. Implement the `renderChart` function in `getChartContext` as shown in this example:  
+
+```jsx
+...
+ renderChart: (context) => renderChart(context),
+...
+```
+At this point, you will notice that the chart is gone on the link.
```
Comment (ShashiSubramanya) [inline README.md:360]: Not sure what this text refers to: 

`At this point, you will notice that the chart is gone on the link.`

File: README.md (hunk 1 above)
Comment (chetan1507) [reply README.md:14]: Let me add the links.

File: README.md (hunk 3 above)
Comment (chetan1507) [reply README.md:296]: This should be added with a TBD link. It would help build connecting pages.

**Pull request merged**